tests/framework/extensions/ingresses/ingresses.go

```diff
@@ -19,3 +30,46 @@ func AccessIngressExternally(client *rancher.Client, hostname string, isWithTLS
 
 	return result.Ok, err
 }
+
+// CreateIngress will create an Ingress object in the downstream cluster.
+func CreateIngress(client *v1.Client, ingressName string, ingressTemplate networking.Ingress) (*v1.SteveAPIObject, error) {
+	podClient := client.SteveType(pod)
+	err := kwait.Poll(15*time.Second, 5*time.Minute, func() (done bool, err error) {
```

Comment:
Can you use the defaults const that you used in verify.go

Reply:
Good call out, I can definitely do that